### ISSUE TITLE
Keep the require('./styles.css') in the JS file

### ISSUE THREAD

**Issue by amaurymartiny** [michalkvasnicak/babel-plugin-css-modules-transform#69]
Right now this plugin transforms
```javascript
// MyComponent.js
import styles from './MyComponent.css';
// ...
```

into
```javascript
// MyComponent.js
var styles = {
  'layout': 'page__layout___2xRof'
};
// ...
```

I also configured the extractCss option to put the CSS output file right next to the babelified JS output file of each component. However, would it be possible to append a `require('./MyComponent.css)` at the beginning of the output JS? I.e. to something like:
```javascript
// MyComponent.js
require('./MyComponent.css');
var styles = {
  'layout': 'page__layout___2xRof'
};
// ...
```

**Comment by amaurymartiny** [comment]
Ha, just saw the PR #65

**Comment by michalkvasnicak** [comment]
@amaurymartiny released as `1.3.0`. Just use `keepImport: true`.
